Paws4Me/Paws4Me/Controllers/Logic/SignInViewController.swift

```diff
@@ -11,20 +11,27 @@ protocol Validation {
 }
 
 class SignInViewController: UIViewController, Validation {
+    // MARK: - IBOulets
     @IBOutlet weak private var txtUsername: UITextField!
     @IBOutlet weak private var txtPassword: UITextField!
     @IBOutlet weak private var btnSignIn: UIButton!
     @IBOutlet weak private var btnRegister: UIButton!
+
+    // MARK: - Vars/Lets
     private var isLoggedIn = false
     private let bottomLine = CALayer()
```

Comment:
Do you need 1 & 2 or can the first element do the job of both?

Reply:
To be honest after a lot of experimentation and research as well as debugging I found that creating 2 instances of CALayer was best because if I only had one CALayer it would only add 1 line to my view and I required 2 lines, one for username and one for password.

Reply:
I still feel you can use one but I think the shortcomings is the function adding the CALayer to your textboxes lets do one more round of investigation on this

Reply:
You were right 👍🏻

Reply:
You were right 👍🏻